components/camera/rtsp/h264decoder.go

```diff
@@ -0,0 +1,173 @@
+package rtsp
+
+// site:https://github.com/aler9/gortsplib/examples
+// author:Alessandro Ros
```

Comment:
This decoder file comes from the example for decoding h264 in the gortsplib repo.